packages/cache/RELEASES.md

```diff
@@ -0,0 +1,5 @@
+# @actions/cache Releases
+
+### 1.0.0
```

Comment:
Any guideline on how to do beta release? Set the release to 0.0.0?

Reply:
With artifacts, we had a `0.1.0` release: https://github.com/actions/toolkit/blob/master/packages/artifact/RELEASES.md

Reply:
If you set the version to `1.0.0-beta.0` we can push a beta tag using `npm publish --tag beta`. Just make sure whoever is doing the first publish (I can) knows.

Since we are re-using a lot of this code and feel confident about the behavior and interfaces, we can also opt to skip a beta release and just release the package.